build.gradle.kts

```
@@ -53,9 +53,12 @@ subprojects {
     tasks.register<Task>("uploadIfNeeded") {
         val tag = System.getenv("TRAVIS_TAG")
         val branch = System.getenv("TRAVIS_BRANCH")
+        val isPullRequest = System.getenv("TRAVIS_PULL_REQUEST")?.toBoolean()
         if (!tag.isNullOrBlank()) {
+            println("Upload artifacts to Bintray...")
```

```suggestion
            project.logger.lifecycle("Upload to Bintray needed.")
```
Because:
* I think we should start using `project.logger` so that user can disable logs if needed
* I think the log happens at configuration time and not execution time so "Upload artifacts to Bintray..." might be a bit misleading as the upload would happen a bit later ? (Not sure about this one though, if the task is really lazy configured maybe the execution happens right after)

---

Works for me